Assignment4/pablohdez98-21A289/task06.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1r1FPUfdtcJeVk4IHRhy4kUmX-icfnLim
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
```

Comment:
This won't run in Python. You need to comment out line 12. Same in the other tasks